pkg/downloader/manager.go

```diff
@@ -571,7 +571,7 @@ func (m *Manager) findChartURL(name, version, repoURL string, repos map[string]*
 	if err == nil {
 		return
 	}
-	err = errors.Errorf("chart %s not found in %s", name, repoURL)
+	err = errors.Errorf("chart %s not found in %s: %v", name, repoURL, err)
```

Comment:
Instead of `%v` is should be `%s` as we want the error as a string.

Reply:
:+1: Actually I had changed to `errors.Wrapf` but apparently didn't save before committing :facepalm: Any preference there?

Reply:
let's keep the change as minimal as possible. We can address the change to a `WrapF` in a separate PR.